action.yml

```diff
@@ -1,6 +1,6 @@
-name: 'git-version-lite'
```

Comment:
Add just to invoke a change.
But as a side not, yml doesn't require these quotes.